admin_manual/configuration_files/federated_cloud_sharing_configuration.rst

```diff
@@ -8,9 +8,15 @@ file shares between ownCloud servers, in effect creating a cloud of ownClouds.
 You can automatically send an email notification when you create the share, add 
 password protection, allow users to upload files, and set an expiration date.
 
+Federated cloud sharing now works a little differently than in ownCloud 8.0. 
```

Comment:
@nickvergessen Not sure what you mean-- direct sharing is not in the 8.0 UI, and the Federated Sharing section is separate from Sharing in 8.0.

![8-0-fed-share](https://cloud.githubusercontent.com/assets/8388387/8062379/40a163b4-0e84-11e5-828f-c93b47386c50.png)